Card style?

sysBlank opened the pull request:
Something like this?

sysBlank committed:
Card basis

ale-cia reviewed (approved):
That's good. The height is fine, it's a little too wide maybe 270-300 x 180?

sysBlank committed:
Added anonther card and made some changes to the first

sysBlank committed:
Merge remote-tracking branch 'refs/remotes/ale-cia/master'

ale-cia reviewed (approved):
That's good!

sysBlank committed:
Just small text change

sysBlank committed:
Front End (Beta)

I did what I could before sleep, there is still a lot more to change,
fix, add, and test.

ale-cia commented:
Implementing new changes so this card design isn't needed.
